{"name": "Verify User is not able to Login with Invalid credentials", "status": "passed", "description": "Verify User is not able to Login with Invalid credentials", "attachments": [{"name": "log", "source": "12e1e6ea-6bd3-495e-895d-f236b358e227-attachment.txt", "type": "text/plain"}], "start": 1589022160342, "stop": 1589022175179, "uuid": "3a6d6722-92a0-4d00-9590-494545e3b05a", "historyId": "57f7ed73470ce8d4aacc4aec2dc80c63", "testCaseId": "e94f0024c374825736805ac54bd2db4c", "fullName": "tests.home.test_login.TestLogin#test_invalidLogin", "labels": [{"name": "tag", "value": "HCMAT-54574"}, {"name": "severity", "value": "minor"}, {"name": "story", "value": "HCMAT-54574"}, {"name": "parentSuite", "value": "tests.home"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "del1-lhp-N01966"}, {"name": "thread", "value": "16452-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.home.test_login"}]}